snippets/auth/index.py

```diff
@@ -192,6 +192,26 @@ def get_user_by_email():
     print('Successfully fetched user data: {0}'.format(user.uid))
     # [END get_user_by_email]
 
+def bulk_get_users():
+    # [START bulk_get_users]
+    from firebase_admin import auth
+
+    result = auth.get_users([
+        auth.UidIdentifier('uid1'),
+        auth.EmailIdentifier('user2@example.com'),
+        auth.PhoneIdentifier(+15555550003),
+        auth.ProviderIdentifier('google.com', 'google_uid4')
+    ])
+
+    print('Successfully fetched user data:')
+    for user in result.users:
+        print(user)
+
+    print('Unable to find users corresponding to these identifiers:')
+    for uid in result.notfound:
+        print(uid)
```

Comment:
This one's unfortunate. There's no `__str__()`, or anything else that can be printed. You could compare to the input identifiers, but that's too wordy for a snippet. This might be the best we can do (and will automatically become better if we add `__str__()` in the future). No action required.